### PR TITLE
link nbclassic docs, note Lab 4 drop notebook dependency

### PR DESCRIPTION
**Pull request by RRosio**
## References
Updating docs to as mentioned in https://github.com/jupyterlab/jupyterlab/issues/12115

## Code changes

Adds a few sentences in documentation offering links to NbClassic documentation which informs users about using JupyterLab, Notebook and NbClassic. Also, mentions the drop in Notebook dependency as of Lab 4.

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks!

File: docs/source/getting_started/overview.rst [hunk 1]
```diff
@@ -85,4 +85,10 @@ We appreciate feedback on our `GitHub issues page <https://github.com/jupyterlab
 as we evolve towards a stable extension development API.
 
 JupyterLab will eventually replace the classic Jupyter Notebook.
-Throughout this transition, the same notebook document format will be supported by both the classic Notebook and JupyterLab.
+Throughout this transition, the same notebook document format will
```
Comment (jtpio) [inline docs/source/getting_started/overview.rst:88]: > JupyterLab will eventually replace the classic Jupyter Notebook.

Wondering if the sentence above is still valid? With the current state of the projects Notebook 7 will be the one replacing the classic notebook 6.

Comment (jtpio) [reply docs/source/getting_started/overview.rst:88]: But it's true we should indeed still mention JupyterLab 4 will not have the `notebook` dependency.

Comment (RRosio) [reply docs/source/getting_started/overview.rst:89]: Right, I think that sentence could be changed to better reflect the state of projects today. I have made some updates which I thought might fit better, but I would appreciate feedback on them!

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**